### PR TITLE
Script fix

### PR DESCRIPTION
**Pull request by anyaelena**
Fix the old reference to `run_file` and the make log path unique for each datapoint.

### CODE REVIEW

**Comment by anyaelena** [comment]
for discussion. i am going to manually test locally running data point scripts with pat in develop mode.  i also want to take a look at why the related spec was passing with that old run_file code still in place.

File: server/app/jobs/dj_jobs/run_simulate_data_point.rb [hunk 1]
```diff
@@ -476,24 +467,30 @@ def upload_file(filename, type, display_name = nil, content_type = nil)
...
-      if File.file? args_path
-        args = Utility::Oss.load_args args_path
-        @sim_logger.info " args loaded from file #{args_path}: #{args}"
-      end
+      begin
+        dir_path = "#{analysis_dir}/scripts/data_point"
+        #  paths to check for args and script files
+        args_path = "#{dir_path}/#{script_name}.args"
+        script_path = "#{dir_path}/#{script_name}.sh"
+        log_path = "#{dir_path}/#{@data_point.id}.log"
```
Comment (rHorsey) [inline server/app/jobs/dj_jobs/run_simulate_data_point.rb]: Would `"#{analysis_dir}/data_point_#{@data_point.id}/#{script_name}.log"` be workable?

Comment (rHorsey) [reply server/app/jobs/dj_jobs/run_simulate_data_point.rb]: Actually `"#{analysis_dir}/data_point_#{@data_point.id}/run/#{script_name}.log"`

Comment (rHorsey) [reply server/app/jobs/dj_jobs/run_simulate_data_point.rb]: I take it back, the first one is better.

**Pull request merged**